app/controllers/admin_controller.rb

```diff
@@ -1,5 +1,5 @@
 class AdminController < ApplicationController
-  before_filter :require_admin, :except => [:index, :show]
+  before_filter :require_admin, :except => [:index, :show, :edit, :update]
```

Comment:
I think this would potentially allow any user to access /admin/users/:id/edit and update actions.. I would think instead of allowing access at this level, you'd want to allow it on the controller(s) which inherit from AdminController